code/API_definitions/device-reachability-status.yaml

```diff
@@ -320,22 +320,21 @@ components:
           $ref: "#/components/schemas/Device"
 
     ErrorInfo:
-      description: The error object used for error-cases.
       type: object
       required:
+        - message
```

Comment:
Same for this one (https://github.com/camaraproject/Commonalities/issues/515) - we should keep the initial order.